middlewares/authenticate.middleware.ts

```diff
@@ -0,0 +1,35 @@
+import { NextFunction, Request, Response } from "express";
+import { errorResponse } from "../util/response.helper";
+import { ResponseMessage } from "../contants/response-message.contant";
+import jwt from 'jsonwebtoken';
+
+
+const AllowedAPIs = [ 
+    'api/account/login', 
+    'api/account/register' 
+];
+
+const authenticate = (request: Request, response: Response, next: NextFunction) => {

```

Comment:
this function can be refactored and improved like this
https://claude.site/artifacts/46f1c3ea-3758-4268-a637-d82b76b1c632